packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -385,8 +463,56 @@ export function SelectPanel({
         },
       } as ItemProps
     })
+
+    setItemsToRender(itemsToRender)
```

Comment:
Could this and the effect below potentially move into just the render function instead of effects or are we experiencing some perf problems when that happens?

Reply:
Yeah we can, I put it in a useEffect as a response to your comment here https://github.com/primer/react/pull/5867#discussion_r2033979938, (also see https://github.com/primer/react/pull/5867#issuecomment-2787771215) so we didn't have to sort on every render 👀